sbt-tasty-mima/src/main/scala/sbttastymima/TastyMiMaPlugin.scala

```diff
@@ -8,6 +8,9 @@ import sbt.librarymanagement._
 import sbt.plugins.JvmPlugin
 
 object TastyMiMaPlugin extends AutoPlugin {
+  // Must stay in sync with TastyMiMaVersion in build.sbt
+  private val TastyMiMaVersion = "0.1.1"
```

Comment:
Can this be done with sbt-buildinfo?

Reply:
It could but I have even better plans for this later on. I think tasty-mima-interface should offer a method that returns its version number, but that's going to be more work that is not critical right now.